utils/dump1090/files/dump1090.config

```diff
@@ -2,49 +2,43 @@ config dump1090 main
 	option disabled '1'
 	option respawn '1'
 	option device_index ''
+	option ppm ''
 	option gain ''
 	option enable_agc '0'
 	option freq ''
+	option lat ''
+	option lon ''
+	option max_range ''
+	option fix '0'
+	option no_fix '0'
 	option ifile ''
 	option iformat ''
 	option throttle '0'
+	option mlat '0'
 	option raw '0'
-	option net '1'
 	option modeac '0'
+	option onlyaddr '0'
+	option metric '0'
+	option gnss '0'
+	option snip ''
+	option net '1'
 	option net_only '0'
 	option net_bind_address ''
 	option net_ri_port ''
 	option net_ro_port ''
+	option net_ro_size ''
+	option net_ro_interval ''
 	option net_sbs_port ''
 	option net_bi_port ''
 	option net_bo_port ''
-	option net_ro_size ''
-	option net_ro_interval ''
-	option net_heartbeat ''
+	option net_stratux_port ''
+    option net_heartbeat ''
```

Comment:
spaces instead of a tab.
The same also once in the init file.

Reply:
my vimrc must be off, should be fixed by the new commit.

Reply:
OK, this time for sure...

Reply:
Third time's a charm (hopefully)